### DISCUSSION TITLE
Web application crash for VosViewer tools

### DISCUSSION THREAD

**Discussion by BSolutionIt** [seinecle/nocodefunctions#3] [General]
Hello, the web application is malfunctioning for the VosViewer tools, it is returning 500. https://nocodefunctions.com/networkconverter/network_format_converter.html

**Comment by seinecle** [comment]
Will look at it first thing today. If you have a file that I can use to reproduce the issue, and that you can share here or to analysis@exploreyourdata.com ?

**Reply by BSolutionIt** [reply]
hi. I sent by email

**Comment by seinecle** [comment]
Hi,
Thank you for the report. This should work now.

**Reply by BSolutionIt** [reply]
Thank you....
